### PR TITLE
Fix travis for Continuous Node PR

### PR DESCRIPTION
**Pull request by yashu-seth**

### CODE REVIEW

**Comment by yashu-seth** [comment]
@ankurankan No relief here too. What should be done?

File: pgmpy/factors/__init__.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from .FactorSet import FactorSet, factorset_product, factorset_divide
 from .CPD import TabularCPD
 from .JointProbabilityDistribution import JointProbabilityDistribution
+from .continuous.ContinuousNode import ContinuousNode
```
Comment (khalibartan) [inline pgmpy/factors/__init__.py:5]: @yashu-seth I think this will fail in python2, try running nosetests for this in python2 in your local system.

Comment (yashu-seth) [inline pgmpy/factors/__init__.py:5]: @khalibartan The tests are passing. Can you please check this too if you have doubts? I don't think there would be a problem.

Comment (khalibartan) [inline pgmpy/factors/__init__.py:5]: @yashu-seth Okay.

**Pull request closed without merging**